server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -501,7 +502,10 @@ private Server startServer(ProjectManager pm, CommandExecutor executor,
             try {
                 final TlsConfig tlsConfig = cfg.tls();
                 if (tlsConfig != null) {
-                    sb.tls(tlsConfig.keyCertChainFile(), tlsConfig.keyFile(), tlsConfig.keyPassword());
+                    try (InputStream keyCertChainInputStream = tlsConfig.keyCertChainInputStream();
+                         InputStream keyInputStream = tlsConfig.keyInputStream()) {
+                        sb.tls(keyCertChainInputStream, keyInputStream, tlsConfig.keyPassword());
+                    }
```

Comment:
Q) Don't We need `ConfigValueConverter` for the passwords in SAML properties?  https://github.com/line/centraldogma/blob/be15612884080517614bb1d468ffb102514a4c30/server-auth/saml/src/main/java/com/linecorp/centraldogma/server/auth/saml/SamlAuthProperties.java#L186-L195

Reply:
It seems there is a TODO for that. https://github.com/line/centraldogma/pull/890/files#diff-1a885e58bb68ec2d7203b8bf406e7ef91752da64cce733cf52c6f0df9e51929cR104
Then, ignore this comment.

Reply:
Just applied the passwords in SAML. PTAL. 😉